firmware/targets/furi_hal_include/furi_hal_speaker.h

```diff
@@ -4,12 +4,22 @@
  */
 #pragma once
 
+#include <furi.h>
+
 #ifdef __cplusplus
 extern "C" {
 #endif
 
 void furi_hal_speaker_init();
 
+void furi_hal_speaker_deinit();
+
+bool __attribute__((warn_unused_result)) furi_hal_speaker_acquire(uint32_t timeout);
```

Comment:
Documentation of ownership methods. It is important to clarify that ownership is given to the thread.

Reply:
Yep, docs please for all methods in furi_hal_speaker